cmd/server_test.go

```diff
@@ -565,7 +570,8 @@ repo-whitelist: "github.com/runatlantis/atlantis"
 require-approval: true
 ssl-cert-file: cert-file
 ssl-key-file: key-file
-ssl-key-file: my-token
+tfe-token: my-token
```

Comment:
I assume this was a botched copy paste, and updated it accordingly

Reply:
good catch! 🍺